packages/pricing-tier-list/docs.mdx

```diff
@@ -0,0 +1,246 @@
+            </svg>,
+          title: 'Development',
+          price: 'Custom Pricing',
+          description: '<p>Managed by your team on your infrastructure of choice. Recommended for teams with custom deployment requirements</p>',
+          cta: {
+            type: 'button',
+            url: '#',
+            title: 'Try it for free',
+            theme: { variant: 'secondary' }
+          }
+        }
+      ]}
+    />
+  `}
+</LiveComponent>
+
+### Props
+<PropsTable props={componentProps} />
+
+### Examples
+
+## With 2 tiers
+<LiveComponent>
+{`
+  <PricingTierList
+    tiers={[
+      {
+        title: 'Development',
+        label: 'Starting at',
+        price: '$0.03',
+        description: '<p>Available as a managed service on HashiCorp Cloud Platform. Recommended for most teams</p>',
+        cta: {
+          type: 'button',
+          url: '#',
+          title: 'Try it for free',
+        }
+      },
+      {
+        title: 'Development',
+        price: 'Custom Pricing',
+        description: '<p>Managed by your team on your infrastructure of choice. Recommended for teams with custom deployment requirements</p>',
+        cta: {
+          type: 'button',
+          url: '#',
+          title: 'Contact sales',
+          theme: { variant: 'secondary' }
+        }
+      }
+    ]}
+  />
+`}
+</LiveComponent>
+
+## With 3 tiers
+<LiveComponent>
+{`
+  <PricingTierList
+    tiers={[
+      {
+        title: 'Development',
+        label: 'Starting at',
+        price: '$1.578',
+        consumption: 'per hour',
+        description: '<p>Clusters designed to scale with the demand of running production workloads.</p>',
+        cta: {
+          type: 'textLink',
+          url: '#',
+          title: 'Try it for free',
+        },
+        supplementaryInfo: '<p>*Price scales with <a>active clients</a></p>'
+      },
+      {
+        title: 'Starter',
+        label: 'Starting at',
+        price: '$1.578',
+        consumption: 'per hour',
+        description: '<p>Clusters designed to scale with the demand of running production workloads.</p>',
+        cta: {
+          type: 'button',
+          url: '#',
+          title: 'Try it for free',
+        },
+        supplementaryInfo: '<p>*Price scales with <a>active clients</a></p>'
+      },
+      {
+        title: 'Standard',
+        label: 'Starting at',
+        price: '$1.578',
+        consumption: 'per hour',
+        description: '<p>Clusters designed to scale with the demand of running production workloads.</p>',
+        cta: {
+          type: 'textLink',
+          url: '#',
+          title: 'Try it for free',
+        },
+        supplementaryInfo: '<p>*Price scales with <a>active clients</a></p>'
```

Comment:
Is it not possible to add more than one line item here in the supplementaryInfo? The use of absolute positioning could lead to layout issues if the text isn't the perfect length shown here in the preview. 
<img width="893" alt="CleanShot 2022-07-12 at 10 51 04@2x" src="https://user-images.githubusercontent.com/825855/178519605-2a6676e8-fa94-4482-8467-166c51d465bf.png">